src/react-components/src/__tests__/components/HelperErrorText-test.tsx

```diff
@@ -4,12 +4,12 @@ import * as Enzyme from "enzyme";
 import * as React from "react";
 import * as Mocha from "mocha";
 
-import HelperErrorText from "../../components/02-molecules/Helpers/HelperErrorText";
+import HelperErrorText from "../../components/01-atoms/Forms/HelperErrorText";
 
 describe("HelperErrorText Test", () => {
-  it("Throws error when HElperErrorText is called with no content", () => {
-    expect(() => Enzyme.mount(<HelperErrorText id="helperTextWithLink"
-    isError={false}/>))
-    .to.throw("Helper and Error Text must have content, or else it should not be used");
+  it("Renders HelperErrorText", () => {
```

Comment:
Since error and helper text are the same thing, tests for the in-error state/not in-error state/in both state?? is that possible/ would be good to see stubbed out here.

Reply:
Added something for Errror state, but HelperErrorText can be helper or error, but not both.  If you have both, you want to use two HelperErrorText objects ... wonder if there's a way to make this clear?

Reply:
I actually don't think you should have both, you're either errored or you're showing helper text. We can illustrate that with a test here.